snuba/settings.py

```diff
@@ -11,7 +11,7 @@
 PORT = 1218
 
 DEFAULT_DATASET_NAME = "events"
-DISABLED_DATASETS: Set[str] = set()
+DISABLED_DATASETS: Set[str] = {"metrics"}
```

Comment:
Ensure this is deployed in production before this PR is.

Reply:
Why does it have to be deployed separately?

Reply:
Because this is for development purposes and prototyping as of now. The storage will not be created in prod any time soon.

Reply:
Right, for a second I thought you intended to deploy this line in the Snuba codebase separately. Now I remembered it's overridden in ops and needs to be changed there.